sphinx_gallery/tests/test_full.py

```diff
@@ -62,3 +64,11 @@ def test_embed_links(sphinx_app):
     assert 'numpy.arange.html' in lines
     # assert '#module-matplotlib.pyplot' in lines
     # assert 'pyplot.html' in lines
+
+
+def test_clean(sphinx_app):
+    subprocess.check_call(
+        [sys.executable, '-msphinx_gallery.clean'], cwd=sphinx_app.srcdir)
```

Comment:
Could you use `sphinx_gallery.clean.main()`. This seems like a good candidate to make codecov happy.